sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -56,12 +52,17 @@
 #include "MacrosMessage.h"
 #include "DYNScenarios.h"
 #include "DYNAggrResXmlExporter.h"
+#include "DYNMPIContext.h"
 
 using multipleJobs::MultipleJobs;
 using DYN::Trace;
 
 namespace DYNAlgorithms {
 
+static DYN::TraceStream TraceInfo(const std::string& tag = "") {
+  return mpi::context().isRootProc() ? Trace::info(tag) : DYN::TraceStream();
```

Comment:
TraceInfo here or Trace::info ?

Reply:
This is the body of TraceInfo method and thus here we use Trace::info